cmd/cloud.go

```
@@ -672,6 +672,215 @@
+	tui.ShowSpinner("fetching projects ...", action)
+	if len(projects) == 0 {
+		fmt.Println()
+		tui.ShowWarning("no projects found")
+		tui.ShowBanner("Create a new project", tui.Text("Use the ")+tui.Command("new")+tui.Text(" command to create a new project"), false)
+		return ""
+	}
+	var options []tui.Option
+	for _, p := range projects {
+		desc := p.Description
+		if desc == "" {
+			desc = "No description provided."
```

golangci-lint (1.64.8)

ineffectual assignment to desc

(ineffassign)